src/components/Modal/OptionApproval.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
Since this is `daoModule` related, maybe this should not be here? (In the general `Modal` folder) - Please let me know if you think this should be moved

Reply:
Hey yes that would be better to have it within the Plugin folder in `/Plugin/DaoModule/Modal/OptionApproval.vue`, in future there might be much more plugins, it's better if each of them have a scoped folder so we don't get too confused.